### ISSUE TITLE
Overfitting on Waymo data set?

### ISSUE THREAD

**Issue by mariya12290** [open-mmlab/mmdetection3d#550]
Hey Admin and all,

I am trying to train Second on waymo data set, but model is getting over fit on waymo data set. Used hyper parameters and model architecture are both same as for kitti data set.

But from observation, waymo data set is sequence, where as kitti is random. So I randomly feeding the data into the network in both data set cases. Still model is overfitting just on waymo.

I am doing the batch normalization, instead of dropout and regularization. From my understanding, batch normalization gives same effect as regularization.

More info about data set and model:

    * Size of data set is almost same around 6k in both kitti and waymo?
    * But, target instance number is different, 5k in kitti and 30k in waymo. 800 and 5k in valid and test respectively.

Here target is pedestrian.

Could someone help me with this problem?

Thanks in advance

**Comment by Tai-Wang** [comment]
We support SECOND both on KITTI and Waymo, please refer to [here](https://github.com/open-mmlab/mmdetection3d/tree/master/configs/second) for configs. There exist differences between these two models in terms of hyper parameters and architectures. What we provide has been after finetuning.

I don't clearly get what your 'overfit' refers to. Although I don't try with `load_interval > 5` before, I did observe some improvements when further using the entire dataset to train the model, so I guess it is not serious overfitting if you use suitable hyper parameters.
